pkg/commons-node/spec/which-spec.js

```diff
@@ -0,0 +1,68 @@
+'use babel';
+/* @flow */
+
+import which from '../which';
+
+describe('which', () => {
+  describe('on windows', () => {
+    const real_platform: string = process.platform;
+    const eol = '\r\n';
+    let os = require('os');
+    const real_eol = os.EOL;
+    beforeEach(() => {
+      Object.defineProperty(process, 'platform', {value: 'win32'});
+      os.EOL = eol;
+    });
+    afterEach(() => {
+      Object.defineProperty(process, 'platform', {value: real_platform});
+      os.EOL = real_eol;
+    });
+
+    it('calls where on Windows', () => {
+      let checkOutput: JasmineSpy = spyOn(require('../process'), 'checkOutput').andReturn('hello');
```

Comment:
You're spying on `checkOutput` on every test and just changing what it returns. To remove boilerplate in each test, and properly unspy, I'd prefer to see, in a `beforeEach` in the top-level `describe`:

```
checkOutput = spyOn(require('../process'), 'checkOutput').andCallFake(() => checkOutputReturn);
```

with `checkOutput` and `checkOutputReturn` at the top.

Then unspy it in the `afterEach`.

Then in each test just set `checkOutputReturn`.